### PR TITLE
Adding SPIRE health check status UI

### PR DESCRIPTION
**Pull request by mamy-CS**
- Addresses https://github.com/spiffe/tornjak/issues/156

### CODE REVIEW

**Review by maia-iyer — COMMENTED** [review]
Tested and it works overall! I had some nit comments just about deleting prints and dead code. 

Also, I was curious - I was able to stop the Tornjak backend api and get the `Inactive` message. However, this means this UI component does not differentiate between Tornjak backend failure (ECONNREFUSED) and SPIRE failure. I'm not sure how one would do that in the UI - maybe there should be another error message type about Tornjak api error? then again, I'm not sure if the Tornjak errors are verbose enough to differentiate. 

What do you think about this thought? Is this fair game for this PR? or does the backend need to be updated to help this?

File: tornjak-frontend/src/components/style.css [hunk 1]
```diff
@@ -315,8 +315,8 @@
     display:inline-block;
   }
 
-  .edit_entries_button {
-  } 
+  /* .edit_entries_button {
+  }  */
```
[the remark this answers is not in the document]
Comment (mamy-CS) [reply tornjak-frontend/src/components/style.css:319]: done!

File: tornjak-frontend/src/components/tornjak-api-helpers.tsx [hunk 2]
```diff
@@ -210,7 +231,7 @@ class TornjakApi extends Component<TornjakApiProp, TornjakApiState> {
     if (serverInfo.plugins === undefined) {
       return
     }
-    console.log(serverInfo.plugins)
+    //console.log(serverInfo.plugins)
```
[the remark this answers is not in the document]
Comment (mamy-CS) [reply tornjak-frontend/src/components/tornjak-api-helpers.tsx:234]: done!

File: tornjak-frontend/src/components/spire-health-check.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  componentDidMount() {
+    console.log("SPIRE HEALTH CHECK TIME SET TO:", spireHealthCheckTime, "seconds")
+    this.startTimer();
+    this.TornjakApi.spireHealthCheck(this.props.spireHealthCheckFunc, this.props.spireHealthCheckingFunc);
+  }
+
+  componentDidUpdate(prevProps: SpireHealthCheckProp, prevState: SpireHealthCheckState) {
+    if (prevState.timer !== this.state.timer) {
+      // timer ended, check status of SPIRE
+      console.log('Checking Status of SPIRE...');
```
[the remark this answers is not in the document]
Comment (mamy-CS) [reply tornjak-frontend/src/components/spire-health-check.tsx:49]: done!

**Comment by mamy-CS** [comment]
> 
I had significant thoughts about this too, I think should be sufficient for this PR. Since the 404 network error when backend fails lets you know that it's an API error. But other options we can do are leaving the SPIRE health check as a loading state and having a more detailed error message for the API error.

**Review by maia-iyer — APPROVED** [review]
LGTM - I agree with the idea of separating the Tornjak backend 404 error; we can make an issue and do that next PR.

**Pull request merged**